<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>chat</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/logo.png</normaloff>icons/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color:#136da7;
}</string>
  </property>
  <widget class="QPushButton" name="msgSend">
   <property name="geometry">
    <rect>
     <x>727</x>
     <y>343</y>
     <width>69</width>
     <height>53</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#msgSend {
	background-color: #c4c4c4;
}</string>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="msgInput">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>343</y>
     <width>428</width>
     <height>53</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#msgInput {
	background-color:white;
}</string>
   </property>
   <property name="documentTitle">
    <string/>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="msgBrowser">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>4</y>
     <width>500</width>
     <height>337</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget {
 color: black;
 background-color:white;
 background-image: url(icons/bg1_1.jpg)  ;
  background-attachment: fixed;
}

QListWidget::item {
 border: 0px solid gray;
 background-color: transparent;
}

QListWidget::item:hover {
 background-color: transparent;
}

QListWidget::item:selected {
 background-color: transparent;
border: none;
}


</string>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QListWidget" name="contactList">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>281</width>
     <height>392</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#contactList {
	background-color:white;
}</string>
   </property>
  </widget>
  <zorder>msgInput</zorder>
  <zorder>msgSend</zorder>
  <zorder>msgBrowser</zorder>
  <zorder>contactList</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
